pkg/virtual/options/options.go

```diff
@@ -33,28 +36,54 @@ import (
 const virtualWorkspacesFlagPrefix = "virtual-workspaces-"
 
 type Options struct {
+	Syncer                 *synceroptions.Syncer
 	APIExport              *apiexportoptions.APIExport
 	InitializingWorkspaces *initializingworkspacesoptions.InitializingWorkspaces
+	Admission              *genericapiserveroptions.AdmissionOptions
 }
 
 func NewOptions() *Options {
-	return &Options{
+	o := &Options{
+		Syncer:                 synceroptions.New(),
```

Comment:
this looks misplaced. It was moved into the tmc dir in its own options struct.

Reply:
Maybe rebase error.

Reply:
Yeah, that's a rebase error